src/pages/Run/index.tsx

```diff
@@ -84,28 +72,7 @@ export default function Run() {
           date
         </FilterLink>
       </div>
-      <ButtonGroup style={{ display: "flex", justifyContent: "center" }}>
-        <Button
-          onClick={() => {
-            setParams({ status: null });
-          }}
-          variant={params.status ? "outlined" : "contained"}
-        >
-          All
-        </Button>
-        {statuses.filter(item => data?.results[item]).map((item) => (
-          <Button
-            key={item}
-            onClick={() => {
-              setParams({ status: item });
-            }}
-            variant={params.status === item ? "contained" : "outlined"}
-          >
-            {statusLabels[item]}
-          </Button>
-        ))}
-      </ButtonGroup>
```

Comment:
Why are we removing this part which added the job-status-count-summary buttons? 
<img width="382" alt="image" src="https://github.com/ceph/pulpito-ng/assets/50358181/38613f69-ca99-4126-9f7e-284bb02411ad">

Reply:
Good question! That was a bit of a hack, and MRT lets us do this instead:
<img width="268" alt="selectstatus" src="https://github.com/ceph/pulpito-ng/assets/58307/69cb2292-678d-43c2-94d6-f77eaf5f6688">

Reply:
Oh that's pretty neat! 🚀